Backpack/Snackbar/Classes/BPKSnackbar.m

```diff
@@ -148,39 +177,36 @@ - (id)init {
 }
 
 - (void)setUpSnackbarWithTitle:(NSString *)title
-                   buttonTitle:(NSString *_Nullable)buttonTitle
-                    buttonIcon:(UIImage *_Nullable)buttonIcon
+                          text:(NSString *_Nullable)text
+                        button:(BPKSnackbarButton *)button
                       leftIcon:(UIImage *_Nullable)leftIcon
                       duration:(BPKSnackbarDuration)duration
                 viewController:(UIViewController *)viewController
-                      delegate:(id<BPKSnackbarProtocol> _Nullable)delegate {
+                      delegate:(id<BPKSnackbarDelegate> _Nullable)delegate {
     self.titleLabel.text = title;
+    self.textLabel.text = text;
     self.delegate = delegate;
     self.snackbarView.backgroundColor = BPKColor.skyBlueShade02;
```

Comment:
Can we make this change to match the Android implementation for DM?

Reply:
I'm sunnure what you mean

Reply:
The snackbar background should be white on DM, right? or we should change Android's implementation to match ours.

Reply:
Yeah I guess we should figure that out but I'd suggest we call that out of scope for this PR